Source/WebCore/page/FrameView.h

```diff
@@ -269,6 +269,10 @@ class FrameView final : public ScrollView {
     void updateCompositingLayersAfterScrolling() final;
     static WEBCORE_EXPORT bool scrollRectToVisible(const LayoutRect& absoluteRect, const RenderObject&, bool insideFixed, const ScrollRectToVisibleOptions&);
 
+    bool requestStartKeyboardAnimation(const KeyboardScroll&) final;
+
+    bool requestFinishKeyboardAnimation(bool immediate) final;
```

Comment:
Remove the blank line above, and rename to `requestFinishKeyboardScrollAnimation`. Is it really "finish", or is it more like "stop"? I think stop would be better.